Example/PrimerSDK/MerchantCheckoutViewController.swift

```diff
@@ -470,17 +274,35 @@ extension MerchantCheckoutViewController: PrimerDelegate {
             )
         )
         
-        requestClientSession(requestBody: clientSessionRequestBody, completion: completion)
+        let networking = Networking()
+        networking.requestClientSession(requestBody: clientSessionRequestBody) { (clientToken, err) in
+            if let err = err {
+                print(err)
+                let merchantErr = NSError(domain: "merchant-domain", code: 1, userInfo: [NSLocalizedDescriptionKey: "Failed to fetch client token"])
```

Comment:
Was wondering whether to move all these hardcoded strings into a separate file...
literally polishing the SDK and having a unique source of truth `Strings.swift`
Even though this is the Example - I think it's good to provide consistency in "the way we code" between the SDK and Example app

For later...but worth to keep in mind